src/cli/run.ts

```diff
@@ -129,13 +129,16 @@ export const runTask = async (
   sys?: d.CompilerSystem
 ): Promise<void> => {
   const logger = config.logger ?? createLogger();
+  const rootDir = config.rootDir ?? '/';
+  sys = sys ?? config.sys ?? coreCompiler.createSystem({ logger });
```

Comment:
We have to do this convoluted thing here because we need access to a `CompilerSystem` below to use `sys.platformPath`. I ran into a bunch of issues with just doing `import { join } from 'path'` with treeshaking, module resolution, and whatnot, so I think this is the best option.

Reply:
oh good point! I haven't done any testing, but I think a little defensive programming here couldn't hurt. I'll do something like this probably:

```ts
const configSys = sys ?? config.sys ?? coreCompiler.createSystem({ logger });
```

and then

```ts
{ ..., sys: configSys, ... }
```

below